share/spice/etsy/etsy.js

```diff
@@ -0,0 +1,57 @@
+(function (env) {
+    "use strict";
+
+    var getPrice = function(currencyCode, price) {
+        if (currencyCode == "USD") {
+            return "$" + price;
+        } else {
+            return price + " " + currencyCode;
+        }
+    }
+
+    env.ddg_spice_etsy = function(api_result) {
+
+        if (!api_result || api_result.results.length == 0) {
+            return Spice.failed('etsy');
+        }
+
+        var endpoint = (
+            'https://www.etsy.com/search?q=' +
+            api_result.params.keywords.replace(" ", "+")
+        );
+
+        Spice.add({
+            id: "etsy",
+
+            // Customize these properties
+            name: "Products",
+            data: api_result.results,
+            meta: {
+                sourceName: "Etsy",
+                total: api_result.results.length,
+                sourceUrl: endpoint,
+                searchTerm: api_result.params.keywords,
+            },
+            normalize: function(item) {
+                var feedback_info = item.User.feedback_info;
+                var image = item.Images[0];
+                var shop = item.Shop;
+                var reviewUrl = "https://www.etsy.com/shop/" + shop.shop_name + "/reviews";
+                return {
+                    img: image.url_170x135,
```

Comment:
Looks like we're missing the `url` property? 

We should have one that sends the user to product page